advanced/Scripts/blacklist.sh

```diff
@@ -53,8 +50,8 @@ HandleOther(){
 
 PopBlacklistFile(){
 	#check blacklist file exists, and if not, create it
-	if [[ ! -f ${blacklist} ]];then
-  	  touch ${blacklist}
+	if [[ ! -f ${blacklistFile} ]];then
```

Comment:
These need to be in the format of  "${blacklistFile}" so that bash doesn't split on space if the filename contains a space.